zerver/openapi/zulip.yaml

```diff
@@ -9128,6 +9128,43 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+    delete:
+      operationId: deactivate-custom-emoji
+      summary: Deactivate custom emoji
+      tags: ["server_and_organizations"]
+      description: |
+        This endpoint is used to [deactivate a custom emoji](https://zulip.com/help/custom-emoji#deactivate-custom-emoji) from the user's
+        organization.
+
+        Note: Deactivated emoji will still be visible in old messages.
```

Comment:
Let's just make this a sentence, "Note that deactivated emoji ...".

Also, what happens to reactions, stream descriptions or user statuses that have the custom emoji when it's deactivated? Do they also remain like existing messages? It would be good to clarify that here.

Also, we should add a sentence, maybe before this about who can deactivate custom emoji. I believe it's admins can deactivate any custom emoji and other users can only deactivate custom emoji that they uploaded.

Reply:
Yeah so the deactivated emoji remains visible at all places so extended that sentence to include all other things.
Also, added the sentence for clarifying who can deactivate the custom emoji.